src/types.ts

```diff
@@ -10,14 +10,21 @@ export interface AuthorizationRequest {
     prefix?: string;
 }
 
+export type TdsDriverName = 'tedious' | 'msnodesqlv8';
+
 export interface IDbConnection {
     user?: string;
     password?: string;
     server?: string;
     host?: string;
     database?: string;
     schema?: string;
-    driver?: 'tedious' | 'msnodesqlv8';
+    driver?: TdsDriverName; // this field is deprecated; pass imported driver to getDb, etc
```

Comment:
I think it would be better to use `ITdsDriver` type here instead.
By doing so, `IDbConnection` will include driver, eliminating the need for the `tdsDriver` parameter in `utils.getDb`.

Reply:
I originally did that, but then I saw that it would be stringified as part of the cache key. Perhaps that's not a problem, or we could mask that field before the config is stringified.

Reply:
I handled this by removing `driver` from the config before stringifying the cache key.